pilot/pkg/networking/core/v1alpha3/configgen.go

```diff
@@ -28,3 +29,8 @@ func NewConfigGenerator(plugins []plugin.Plugin) *ConfigGeneratorImpl {
 		Plugins: plugins,
 	}
 }
+
+// Called when mesh config is changed.
+func (configgen *ConfigGeneratorImpl) MeshConfigChanged(mesh *meshconfig.MeshConfig) {
+	rebuildCachedListeners(mesh)
```

Comment:
maybe better call resetAccessLog?

Reply:
Thought of making this method generic so that in future other cached filters can be reset. Should I call `resetCachedListeners` instead of `rebuildCachedListeners` ?

Reply:
Actually we donot cache listener at all. So i am confused by it

Reply:
Ok. I can change to resetAccessLog or resetCachedConfig? or any other better generic name you have -  I am fine with it

Reply:
I think its good to be generic. There are other things we should cache as well, like tracing config, etc

Reply:
renamed it to `resetCachedListenerConfig` -  so that if we do it for clusters/routes we can add additional methods like `resetCachedClusterConfig` etc.